<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>colorReduce</class>
 <widget class="QWidget" name="colorReduce">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>804</width>
    <height>627</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>colorReduce</string>
  </property>
  <widget class="QPushButton" name="m_openVedio">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>打开摄像头</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_reduce">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>处理图像</string>
   </property>
  </widget>
  <widget class="QSlider" name="m_colorValue">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>130</y>
     <width>19</width>
     <height>361</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="m_imageShow">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>640</width>
     <height>480</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="m_getFrame">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>75</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>获取一帧</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_red">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>50</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>R-V</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_green">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>50</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>G-S</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_blue">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>50</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>B-H</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_redValueText">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>80</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_greenValueText">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>80</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_blueValueText">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>80</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="m_closeCamera">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>75</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭摄像头</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_reduceClose">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>10</y>
     <width>75</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>取消处理</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>10</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>阈值</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_thresholdText">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>10</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="m_msg">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>540</y>
     <width>721</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="m_channelR">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>540</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>channel-R-V</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_channelG">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>570</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>channel-G-S</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_channelB">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>600</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>channel-B-H</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_RGBButton">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>10</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>RGB</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_HSVButton">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>10</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>HSV</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
